ert_shared/services/_base_service.py

```diff
@@ -53,6 +54,15 @@ def local_exec_args(script_args: Union[str, List[str]]) -> List[str]:
     return [sys.executable, str(Path(__file__).parent / script), *rest]
 
 
+def delete_connection_info(service_name: str) -> None:
+    """
+    Ensure that the JSON connection information file is deleted
+    """
+    file = Path(f"{service_name}_server.json")
+    if file.exists():
+        file.unlink()
```

Comment:
Maybe something like to unregister?:
```suggestion
        file.unlink()
    atexit.unregister(delete_connection_info)
```

Reply:
nop, no difference